pkg/services/contexthandler/contexthandler.go

```diff
@@ -282,7 +282,8 @@ func (h *ContextHandler) initContextWithAnonymousUser(reqContext *contextmodel.R
 				reqContext.Logger.Warn("tag anon session panic", "err", err)
 			}
 		}()
-		if err := h.anonDeviceService.TagDevice(context.Background(), httpReqCopy); err != nil {
+
+		if err := h.anonDeviceService.TagDevice(context.Background(), httpReqCopy, anonymous.AnonDevice); err != nil {
```

Comment:
Question: do we also need to tag the authed devices for the old (non-auth-broker) auth flow?

Reply:
Auth broker is now enabled by default for every deployment scenario, auth context handler can be removed :smile: